src/query/interpret/awesome_memgraph_functions.cpp

```diff
@@ -1174,6 +1177,41 @@ TypedValue Duration(const TypedValue *args, int64_t nargs, const FunctionContext
   MapNumericParameters<Number>(parameter_mappings, args[0].ValueMap());
   return TypedValue(utils::Duration(duration_parameters), ctx.memory);
 }
+
+std::function<TypedValue(const TypedValue *, int64_t, const FunctionContext &)> UserFunction(
```

Comment:
Why make primitive argument `const`?

Reply:
Since you are not changing the passed value in the given scope

Reply:
This might not be the best, but because it doesn't affect the functionality/behavior of the function, I think we can merge this. The `const` qualifier for primitive by value parameters is only important in the definition, not in the declaration. In other words the `const` for primitive by value parameters is important only at the implementation of the function, in this case in the lambda.